### ISSUE TITLE
Error in Version 0.7.0:  The method attributesToString() is undefined for the type 

### ISSUE THREAD

**Issue by ssardina** [sarl/sarl#815]
#### Issue description

When I compiled my working SARL system with Version 0.7.0 I get this error on every event that has some constructor:

**The method attributesToString() is undefined for the type Event	E_AgentPercept.sarl**

I suspect this is related to https://github.com/sarl/sarl/issues/748

#### Steps to reproduce the issue

1. Just have this event:

```
event E_SpawnAgent {
    var agentName : String

    new(name : String) {
        this.agentName = name;
    }
}

```
 
2. 
3. 

#### What's the expected result?

-

#### What's the actual result?

-

#### Additional details / screenshot

- SARL version: 0.7.0
- SARL compiler:
  - [] Eclipse compiler without Maven
  - [X] Eclipse compiler with Maven
  - [ ] Maven compiler on the command line
  - [ ] sarlc compiler
- Java JDK version (with the manufacturer name): Oracle
- Operating System:
  - [x] Linux 64bits
  - [ ] Linux 32bits
  - [ ] Windows 64bits
  - [ ] Windows 32bits
  - [ ] MacOS 64bits
- ![Screenshot]()

**Comment by gallandarakhneorg** [comment]
I propose to fix for a minor release 0.7.1, which I'm planning for one or two weeks.

**Comment by gallandarakhneorg** [comment]
I cannot reproduce neither on 0.7 or  0.8.
What is the version of SARL or Janus within the classpath of your application?

**Comment by ssardina** [comment]
mmm I just compiled with mvn on CLI and the error was different and related to the EIS interface .toString() which seems it was deprecated, and somehow worked with SARL 0.6.1 but not with 0.7.0. But it is about the EIS not SARL; I changed the code to a different method and it compiles well.

The error message on the ECLIPSE IDE was not indicating that, it was giving an error at the level of the whole Event class.

So, I think this is not per se related to SARL @gallandarakhneorg , sorry. Thanks and I am closing it. Seems to compile now well.
